### PR TITLE
Refactor worker activity

### PR DESCRIPTION
**Pull request by benrudolph**
@czue man this report is a bit of monster. refactored a bit but still kind of brutal. best reviewed :blowfish:. haven't written any ES tests yet, but going to once i get my other PR in that has some other infrastructure that I'd like to use

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 9 errors:

* corehq/apps/reports/analytics/esaccessors.py, line 71 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/reports/standard/monitoring.py, line 1265 - E501 line too long (141 > 115 characters)
* corehq/apps/reports/standard/monitoring.py, line 1271 - E501 line too long (125 > 115 characters)
* corehq/apps/reports/standard/monitoring.py, line 1273 - E501 line too long (125 > 115 characters)
* corehq/apps/reports/standard/monitoring.py, line 1279 - E226 missing whitespace around arithmetic operator
* corehq/apps/reports/standard/monitoring.py, line 1312 - E501 line too long (120 > 115 characters)
* corehq/apps/reports/standard/monitoring.py, line 1333 - E226 missing whitespace around arithmetic operator
* corehq/apps/reports/standard/monitoring.py, line 1340 - E261 at least two spaces before inline comment
* corehq/apps/reports/standard/monitoring.py, line 1340 - E501 line too long (119 > 115 characters)

File: corehq/apps/reports/analytics/esaccessors.py [hunk 1]
```diff
@@ -19,6 +20,64 @@ def convert_to_date(date):
...
+
+    case_query = (CaseES()
+        .domain(domain)
+        .filter(
+            filters.date_range(
+                date_field,
+                gte=datespan.startdate.date(),
+                lte=datespan.enddate.date(),
+            )
+        )
```
Comment (dimagimon) [inline corehq/apps/reports/analytics/esaccessors.py]: E123 closing bracket does not match indentation of opening bracket's line

Comment (benrudolph) [inline corehq/apps/reports/analytics/esaccessors.py]: (ignoring)

File: corehq/apps/reports/tests/test_esaccessors.py [hunk 2]
```diff
@@ -9,13 +9,20 @@
 from corehq.apps.users.models import CommCareUser
 from corehq.apps.groups.models import Group
 from corehq.form_processor.tests.utils import TestFormMetadata
+from casexml.apps.case.models import CommCareCase, CommCareCaseAction
+from casexml.apps.case.const import CASE_ACTION_CREATE, CASE_ACTION_CLOSE
```
Comment (dimagimon) [inline corehq/apps/reports/tests/test_esaccessors.py]: F401 'CASE_ACTION_CLOSE' imported but unused

**Comment by benrudolph** [comment]
Ok @czue I think this is g2g. i ran the report on my master branch and this branch on various filters and am getting the same output.
my branch:
screen shot 2015-12-16 at 6 20 23 pm
master:
screen shot 2015-12-16 at 6 11 38 pm

**Comment by benrudolph** [comment]
this experience has made me want to setup report testing. not sure about the ROI though

**Comment by czue** [comment]
report testing would be great - think as long as we're trending in that direction that may be the best we can do without a huge undertaking

**Comment by czue** [comment]
looks really good. had some questions and i think one blocker on the translations. also I didn't fully grok https://github.com/dimagi/commcare-hq/commit/7b74e2750b2e7863b399360072b175e5dac35a89 so might be good to get someone else to take a quick look at that (am guessing is straightforward, just was an intimidating diff)

**Comment by benrudolph** [comment]
will update thanks. yeah i imagine there are few that would grok what is going on just from the diff. maybe i can walk through that commit with someone tomorrow

**Comment by benrudolph** [comment]
Ok updated, tried to make as many methods static without doing crazy parameter passing through lots of different functions

**Comment by czue** [comment]
alright, did a second pass and this looks good to me. gnarly stuff.

**Pull request merged**